<launch>
  <arg name="device" default="/dev/hydra" />
  <arg name="tf" default="false" />
  <arg name="ms" default="10" />
  <arg name="lambda" default="0.5" />

  <node pkg="razer_hydra" type="driver" name="razer_hydra_driver" output="screen" >
    <param name="device"        type="string" value="$(arg device)" />
    <param name="publish_tf"    type="bool"   value="$(arg tf)" />
    <param name="polling_ms"    type="int"    value="$(arg ms)" />
    <param name="lambda_filter" type="double" value="$(arg lambda)" />
  </node>

</launch>
